scriptmodules/emulators/scummvm.sh

```diff
@@ -62,9 +62,7 @@ function configure_scummvm() {
     mkRomDir "scummvm"
 
     local dir
-    mkUserDir "$home/.local"
     for dir in .config .local/share .cache; do
```

Comment:
The helper won't account for `.cache`?

Reply:
It doesn't but probably shouldnt symlink that - cache should remain in home I guess. We can make the .cache folder though. I'll add it.

Reply:
I now create the .cache folder and no longer symlink anything from the .cache as there is no need imho to move this to the config area (can be regenerated etc if needed)